Unrecognized command or argument '-p:UseSharedCompilation=false' when running on Github build

Version : 2.6.1
I'm receiving this strange error when running on Github Actions runner on a new workflow. Strangely the same build works on an older workflow (with exact same parameters). The only difference is the slightly updated `ubuntu-latest` image the runner is using for one workflow but not the other. The error:
```
/home/runner/.nuget/packages/msbuild.sdk.sqlproj/2.6.1/Sdk/Sdk.targets(242,5): error MSB3073: The command "dotnet "/home/runner/.nuget/packages/msbuild.sdk.sqlproj/2.6.1/Sdk/../tools/net8.0/DacpacTool.dll" build -o "obj/Debug/*****Build.dacpac" -n "*****.Build" -v "1.0.0" -sv Sql140 -i "obj/Debug/***.InputFiles.txt"   -bp AllowSnapshotIsolation=True -dp IncludeCompositeObjects=true --predeploy ../*****/Script.PreDeployment.sql --postdeploy ../*****/Script.PostDeployment.sql --refactorlog ../*****/*****i.refactorlog      " exited with code 1. [/home/runner/work/*****/DB/Build/*****.Build.csproj]
  Using target framework net8.0 to run DacpacTool
  Unrecognized command or argument '-p:UseSharedCompilation=false'.
  Unrecognized command or argument '-p:EmitCompilerGeneratedFiles=true'.
```

For the life of me I can't figure out why this is happening. Any suggestions are welcome. Thank you.

--- a/src/DacpacTool/Program.cs
+++ b/src/DacpacTool/Program.cs
@@ -1,12 +1,15 @@
 ﻿using System;
 using System.CommandLine;
 using System.CommandLine.NamingConventionBinder;
+using System.CommandLine.Parsing;
 using System.Diagnostics;
 using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
+using System.Transactions;
 using Microsoft.SqlServer.Dac;
 using Microsoft.SqlServer.Dac.Model;
+using Microsoft.SqlServer.TransactSql.ScriptDom;
 
 namespace MSBuild.Sdk.SqlProj.DacpacTool
 {
@@ -72,7 +75,8 @@ namespace MSBuild.Sdk.SqlProj.DacpacTool
             var rootCommand = new RootCommand { buildCommand, collectIncludesCommand, deployCommand };
             rootCommand.Description = "Command line tool for generating a SQL Server Data-Tier Application Framework package (dacpac)";
 
-            return await rootCommand.InvokeAsync(args);
+            var processed = rootCommand.Parse(args);
+            return await processed.InvokeAsync();
         }
 
         private static int BuildDacpac(BuildOptions options)

--- a/src/DacpacTool/Program.cs
+++ b/src/DacpacTool/Program.cs
@@ -6,10 +6,8 @@ using System.Diagnostics;
 using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
-using System.Transactions;
 using Microsoft.SqlServer.Dac;
 using Microsoft.SqlServer.Dac.Model;
-using Microsoft.SqlServer.TransactSql.ScriptDom;
 
 namespace MSBuild.Sdk.SqlProj.DacpacTool
 {

